.github/workflows/python-publish.yml

```diff
@@ -22,7 +25,11 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install setuptools wheel twine
+    - name: Test
+      run: |
+        make test
```

Comment:
`make test` seems equivalent to `python setup.py test`, which is now deprecated(https://github.com/pypa/setuptools/issues/1684). 

I get a warning when I run this, see: https://github.com/GenevieveBuckley/dask-image/runs/1340523683?check_suite_focus=true

```
Run make test
python setup.py test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
```

If possible, I'd like us to replace this with something that is less likely to stop working at some future point in time.

Reply:
Thanks. I am +1 for this.

Reply:
```suggestion
        python setup.py test
```